### PR TITLE
MAINTAINERS: add @nicokosi to collaborators

### LINKED ISSUE #5873: Add collaborator: @nicokosi
Hi, @nicokosi! You seem to be enjoying contributing to the tldr-pages project.
You now have had 12 distinct pull requests merged (#5841, #5820, #5818, #5805, #5804, #5797, #5764, #5742, #5693, #5675, #5666, #5662)!
That qualifies you to become a collaborator in this repository, as explained in our [community roles documentation](https://github.com/tldr-pages/tldr/blob/master/COMMUNITY-ROLES.md).

As a collaborator, you will have commit access to the repository.
That means you can merge pull requests, label and close issues, and perform various other maintenance tasks that are needed here and there.
Of course, all of this is voluntary — you're welcome to contribute to the project in whatever ways suit your liking.

If you do decide to start performing maintenance tasks, though, we only ask you to get familiar with the [maintainer's guide](https://github.com/tldr-pages/tldr/blob/master/contributing-guides/maintainers-guide.md).

So, what do you say? Can we add you as a collaborator?

Either way, thanks for all your work so far!

### PR DESCRIPTION
**Pull request by nicokosi**
Closes #5873

### CODE REVIEW

**Comment by navarroaxel** [comment]
Looks like you need to upgrade your fork, do you need help with that?

**Comment by bl-ue** [comment]
I fixed the conflicts via the GH UI.  @nicokosi to update your `main` branch, run this:
```
git checkout main
git fetch upstream main
git rebase upstream/main
git push
```

**Review by sbrl — APPROVED** [review]
Welcome, @nicokosi!

**Review by navarroaxel — APPROVED** [review]

**Review by bl-ue — APPROVED** [review]
Welcome aboard!!

Feel free to merge this PR yourself, @nicokosi 😉

File: MAINTAINERS.md [hunk 1]
```diff
@@ -43,7 +43,9 @@ If you are an owner of the organization, you can see an automated list
 - **Florian Benscheidt ([@Waples](https://github.com/Waples))**:
   [16 April 2021](https://github.com/tldr-pages/tldr/issues/5774) - present
 - **Adam Herst ([@aherst](https://github.com/aherst))**:
-  [21 April 2021](https://github.com/tldr-pages/tldr/issues/5810) - present  
+  [21 April 2021](https://github.com/tldr-pages/tldr/issues/5810) - present
+- **Nicolas Kosinski** ([@nicokosi](https://github.com/nicokosi)):
```
Comment (bl-ue) [inline MAINTAINERS.md:47]: Oh, looks like you misplaced the closing `**`:
```suggestion
- **Nicolas Kosinski ([@nicokosi](https://github.com/nicokosi))**:
```

**Review by marchersimon — APPROVED** [review]

**Pull request merged**